src/study.js

```diff
@@ -80,8 +81,16 @@ export default async function runStudy(devMode) {
             Glean.setUploadEnabled(false);
           }
       });
+      
+      // send the study enrollment ping to the secure analysis environment.
+      const uid = devMode ? "00000000-0000-0000-0000-000000000000" : rally._rallyID;
+      if (!devMode && !uid) {
+        console.error("Rally ID not acquired by study.");
+      }
+      studyEnrollmentMetrics.id.set(uid);
```

Comment:
Mh, is `rally.id.set(rally._rallyID);` ever called somewhere else?

Reply:
Not currently, but it will be. I think I see where you're going with this :) the PR that adds the rally id to the RS01event has not merged yet. Going to figure out how to line these up together and we can set `rally.id.set(rally._rallyID)` once at the beginning of this.

Reply:
Yes :-) That would make a ton of sense and save you some code!